### PR TITLE
Add BinaryenCallIsReturn/BinaryenCallIndirectIsReturn to C/JS API

### PR DESCRIPTION
**Pull request by TerrorJack**

### CODE REVIEW

File: src/binaryen-c.cpp [hunk 1]
```diff
@@ -2112,6 +2112,16 @@ BinaryenExpressionRef BinaryenSwitchGetValue(BinaryenExpressionRef expr) {
   return static_cast<Switch*>(expression)->value;
 }
 // Call
+uint8_t BinaryenCallIsReturn(BinaryenExpressionRef expr) {
```
Comment (dcodeIO) [inline src/binaryen-c.cpp:2115]: Not sure how important this is, but other API methods typically use `int` to denote bools.

**Comment by tlively** [comment]
@TerrorJack Can you merge master into this? The CI we use changed to GitHub actions in the meantime, so the tests won't run unless you do a merge.

**Comment by TerrorJack** [comment]
> This pull request can be automatically merged by project collaborators
> Only those with write access to this repository can merge pull requests.

@tlively

**Comment by dcodeIO** [comment]
Try the following locally, in your `call-is-return` branch:

```
git checkout master
git pull
git checkout call-is-return
git merge master
git push
```

to merge the changes meanwhile pushed to master, so CI can pick up the new configuration here and complete the checks.

**Comment by TerrorJack** [comment]
Ah sorry, I thought it's merging back to `master` (and thus I have no permission). Done rebasing on latest `master`.

**Pull request merged**